app/controllers/admin/userManagement.py

```diff
@@ -65,7 +65,7 @@ def deleteProgramFile():
 
 @admin_bp.route('/admin/updateProgramInfo/<programID>', methods=['POST'])
 def updateProgramInfo(programID):
-    if g.current_user.isCeltsAdmin:
+    if g.current_user.isCeltsAdmin or g.current_user.isProgramManager():
```

Comment:
This logic would allow any program manager update any program, even if they are not program managers for other programs. That is mostly applicable for student program managers. Fix it with program id